test/python/transpiler/test_mappers.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Meta tests for mappers.
+
+The test checks the output of the swapper to a ground truth DAG (one for each
+test/swapper) saved in as a pickle (in `test/python/pickles/`). If they need
+to be regenerated, the DAG candidate is compiled and run in a simulator and
+the count is checked before being saved. This happens with (in the root
+directory):
+
+> python -m  test.python.transpiler.test_mappers
+
+To **add a swapper**: add a new class to this file:
```

Comment:
I had some problems understanding what was “adding a swapper”, if I understood correctly, this is how to setup a test class for passing all the common tests to the a new swapper (possibly in addition to specific ones).

Can we clarify this? For instance:

```
To make a new swapper pass all the common tests, create a new call inside the file path/to/test_mappers.py that:
  * inherits from...
```

Reply:
done in 962363aa